src/components/20-molecules/popup/index.js

```diff
@@ -7,7 +7,7 @@ export * from './popup-mixin';
 /* eslint-disable no-undef */
 const axaPopupVersion = __VERSION_INFO__;
 /* eslint-enable no-undef */
-const commonVersion = axaPopupVersion['axa-popup']['axa-popup'];
+const commonVersion = axaPopupVersion['axa-popup'];
```

Comment:
This has nothing to do with the currency feature directly. But without that change the Unit Tests are failing. @markus-walther can you please check that this change is ok?

Reply:
This change is definitely NOT ok. It leads to definitions like &lt;axa-popup-button-objectObject&gt; ...
Please revert it and adapt the unit tests instead.

Reply:
Reverted and fixed.